style.css

```diff
@@ -174,3 +178,307 @@ body {
+#recent-work-text h1 {
+  width: 327px;
+  height: 97px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  order: 0;
+}
+
+#recent-work-text img {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+  order: 0;
+}
+
+#card-row {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4px 2px;
+  gap: 24px;
+  width: 331px;
+  height: 2972px;
+  order: 1;
+}
+
+.card {
+  width: 327px;
+  height: 474px;
+  position: relative;
+  top: 4px;
+  left: 2px;
+}
+
+.card-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 16px;
+  position: absolute;
+  width: 327px;
+  height: 222px;
+  left: calc(50% - 327px / 2);
+  bottom: 0;
+  background-color: #fff;
+  border-radius: 8px;
+  border: 1px solid #d0d9d4;
+}
+
+.project-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.tags1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 305px;
+  height: 32px;
+}
+
+.inner-tag1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  height: 32px;
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+.inner-tag1 h5 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+.img-holder {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  z-index: -1;
+}
+
+.see-project {
+  width: 305px;
+  height: 69px;
+  position: relative;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  position: absolute;
+  left: calc(50% - 122px / 2 + 0.5px);
+  bottom: 8px;
+  background: #36b37f;
+  border: 0;
+  border-radius: 4px;
+}
+
+.btn-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+#about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 68px;
+  position: absolute;
+  width: 375px;
+  height: 1625px;
+  left: 0;
+  top: 3927px;
+}
+
+#about-me-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 351px;
+  height: 333px;
+}
+
+.btn-resume {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  background: #36b37f;
+  border: 0;
+  border-radius: 4px;
+  margin-top: 5px;
+}
```

Comment:
- [ ] Kindly try to add some engagement effects to your buttons to make the more interactive to users like `hover ` or `pressed` effects can be awesome 👍